package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Serverless ESBuild example using Typescript",
   "main": "handler.js",
   "scripts": {
-    "deploy-local": "docker-compose exec -u node -w /app offline /serverless/node_modules/serverless/bin/serverless.js deploy --log --profile localstack --stage dev",
-    "invoke-local": "docker-compose exec -u node -w /app offline /serverless/node_modules/serverless/bin/serverless.js invoke --log --profile localstack --stage dev --function ",
-    "invoke-local-stepf": "docker-compose exec -u node -w /app offline /serverless/node_modules/serverless/bin/serverless.js invoke stepf --log --profile localstack --stage dev --name ",
+    "jest-test-debug": "export $(cat ./debug/.dev.env | xargs) && node --inspect=0.0.0.0:9229 ./node_modules/.bin/jest --runInBand",
```

Comment:
Why do we need to pass all the user environment variables to serverless?

Reply:
Hum, thinking of this repo as the template only, I think it's actually good that we commit `.vscode` folder. Probably I was over thinking when I made this. I have a habit of adding my own config into that folder. That's why I tried to prevent committing it.

Regarding passing user's environment variable, they are flags for enable debugging, which are stored in `./debug/.dev/env'. These variables are:
```
SLS_DEBUG=*                                                     # Tell serverless to run in debug mode
MINIFY=false                                                        # Tell serverless-esbuild to disable minify
SOURCEMAP=true                                              # Tell serverless-esbuild to enable sourcemap

AWS_LOCALSTACK=172.20.0.100                     # Tell our lambda function where localstack is
```